### ISSUE TITLE
Form Designer

### ISSUE THREAD

**Issue by TiKevin83** [PiranhaCMS/piranha.core#628]
Issue 467 mentions a number of potential CMS-oriented packages that could be delivered alongside piranha core.  I'm particularly interested in the idea for a form designer.  I'd be open to collaborating on a package for it with anyone else interested.  It may have a sub-dependency for a notification system, eg sending an email when a form is submitted.

**Comment by ksahmed** [comment]
I would be interested @TiKevin83  in contributing on a forms designer .. i am really struggling for 2 days to build contact us form :)

**Comment by shapeh** [comment]
I agree with having a forms builder. This will make adoption easier.

**Comment by gwhitdev** [comment]
> I would be interested @TiKevin83 in contributing on a forms designer .. i am really struggling for 2 days to build contact us form :)

What has your approach been so far?

**Comment by gwhitdev** [comment]
> I would be interested @TiKevin83 in contributing on a forms designer .. i am really struggling for 2 days to build contact us form :)

@ksahmed what has your approach been so far?

**Comment by gwhitdev** [comment]
@TiKevin83 have you made any progress on this at all? I'd be interested in assisting with this.

**Comment by TiKevin83** [comment]
I haven't sorry, I've been looking more at SSG which has drawn me away from interest in forms
